packages/typespec-client-generator-core/test/types.test.ts

```diff
@@ -1080,6 +1081,27 @@ describe("typespec-client-generator-core: types", () => {
       strictEqual(enumType.values[0].name, "ARename");
       strictEqual(enumType.values[1].name, "BRename");
     });
+
+    it("versioned enums", async () => {
+      await runner.compile(
+        `
+        @versioned(Versions)
+        @service({
+          title: "Widget Service",
+        })
```

Comment:
```suggestion
        @service
```

nit, doesn't seem necessary for this test